api/v2/performanceprofile_types.go

```diff
@@ -73,8 +73,7 @@ type CPU struct {
 	// except the reserved CPUs. In order to guarantee that your workload will run on the isolated CPU:
 	//   1. The union of reserved CPUs and isolated CPUs should include all online CPUs
 	//   2. The isolated CPUs field should be the complementary to reserved CPUs field
```

Comment:
this text is slightly misleasing and we need to improve it:
1. is not true: we had cases on which isolated and reserved were non-overlapping subsets of the online CPUs, and this is supported and should work
2. this is hard to parse, probably simpler to say that the two set MUST not overlap (and nowadays PAO validates that)
I'll file a PR to fix the comment.